cmd/src/actions_create.go

```diff
@@ -7,27 +7,29 @@ import (
 	"os"
 )
 
-const actionDefinitionTemplate = `{
-  "$schema": "https://raw.githubusercontent.com/sourcegraph/src-cli/master/schema/actions.schema.json",
-  "scopeQuery": "",
-  "steps": [
-  ]
-}
+const actionDefinitionTemplate = `$schema: https://raw.githubusercontent.com/sourcegraph/src-cli/master/schema/actions.schema.json
+
+scopeQuery: ""
+
+steps:
+- type: command
+  args:
+  - /bin/sh
```

Comment:
That's not a good example since it's not immediately clear whether the args to `bin/sh` end up on this line or not.